main/main.go

```diff
@@ -66,9 +67,8 @@ func main() {
 	}
 
 	rand.Seed(time.Now().UTC().UnixNano())
-	privKey, err := crypto.GeneratePrivateKey()
-	if err != nil {
-		fmt.Println("Error while generating private key", err)
+	privKey := crypto.GeneratePrivateKey(KEY_DIFF)
```

Comment:
the previous version with explicit error is better, revert to it

Reply:
Done